src/main/java/org/apache/commons/collections4/iterators/ListIteratorWrapper.java

```diff
@@ -204,7 +204,7 @@ public int previousIndex() {
 
     /**
      * Removes the last element that was returned by {@link #next()} or {@link #previous()} from the underlying collection.
-     * This call can only be made once per call to {@code next} or {@code previous} and only if {@link #add()} was not called in between.
+     * This call can only be made once per call to {@code next} or {@code previous} and only if {@link #add(Object)} was not called in between.
```

Comment:
Was this necessary to solve the animal sniffer problems too?

Reply:
@kinow no.
that is yet another error that stop the build.